knowledge-base/how-to-upgrade-an-old-project-to-the-latest-telerik-reporting-version.md

```diff
@@ -0,0 +1,37 @@
+
+Details how to download MSI files and upgrade project's items can be found in <a href="http://docs.telerik.com/reporting/installation-installing-from-msi" target="_blank">Installing Telerik Reporting</a> and <a href="http://docs.telerik.com/reporting/installation-upgrading-newer-version" target="_blank">Upgrading Telerik Reporting Projects</a>.  
+
+### Some important changes during the years:  
+- Reports should be wrapped in a <a href="https://docs.telerik.com/reporting/report-sources" target="_blank">report source object</a> in order to be passed to a report viewer, SubReport or action - <a href="/support/kb/reporting/details/how-to-migrate-your-project-to-utilize-the-new-reportsource-objects" target="_blank">How to migrate your project to utilize the new ReportSource objects</a>.
+- As of <a href="/support/whats-new/reporting/release-history/telerik-reporting-q1-2013" target="_blank">Q1 2013</a> there are removed several obsolete properties and namespaces.
+- As of <a href="/support/whats-new/reporting/release-history/telerik-reporting-q2-2013" target="_blank">Q2 2013</a> Telerik Reporting is built against .NET4 framework Full Profile, which is supported in Visual Studio 2010+ versions. Also the old Chart item is made obsolete. Its successor is the native reporting <a href="https://docs.telerik.com/reporting/graphoverview" target="_blank">Graph item</a>.
+- As of <a href="/support/whats-new/reporting/release-history/telerik-reporting-q3-2013" target="_blank">Q3 2013</a> there is a new web viewer - the <a href="https://docs.telerik.com/reporting/html5-report-viewer-system-requirements" target="_blank">HTML5 Report Viewer</a>, which is a client-side widget served by a running <a href="https://docs.telerik.com/reporting/telerik-reporting-rest-conception" target="_blank">Reporting REST service</a>, dedicated to work with the viewer.
+- As of <a href="/support/whats-new/reporting/release-history/telerik-reporting-q1-2014" target="_blank">Q1 2014</a> the WPF and Silverlight ReportViewer controls use only implicit styles.
+- As of <a href="/support/whats-new/reporting/release-history/telerik-reporting-q3-2015-(version-9-2-15-930)" target="_blank">Q3 2015</a> the old ASP.NET WebForms ReportViewer is obsolete.
+- As of <a href="http://docs.telerik.com/reporting/upgrade-path-2016-r3" target="_blank">R3 2016</a> the ObjectDataSource component requires additional settings (security improvement).
+- As of <a href="/support/kb/reporting/deploy-to-web-desktop-cloud/details/telerik-reporting-r3-2016---upgrade-issues-with-objectdatasource-component" target="_blank">R3 2016</a> changes on report items definitions in events are not accepted.
+
+## Additional Resources

```

Comment:
As per KB standards, this section should be called ##See Also - https://wiki.progress.com/pages/viewpage.action?spaceKey=DTS&title=Knowledge+Base+Content+Standards

Reply:
Done